src/components/authorization/policies/by-role/consultant.policy.ts

```diff
@@ -22,6 +22,13 @@ import { inherit, member, Policy, Role } from '../util';
   r.Partnership.when(member).read,
   r.PeriodicReport.read,
   r.Product.read,
+  r.ProgressReport.when(member).read.children((c) =>
+    [c.teamNews, c.communityStories, c.highlights].flatMap((it) => it.read),
+  ),
+  r.ProgressReportVarianceExplanation.when(member).read,
+  r.ProgressReportTeamNews.read.specifically((p) => p.responses.read),
+  r.ProgressReportCommunityStory.read.specifically((p) => p.responses.read),
+  r.ProgressReportHighlight.read.specifically((p) => p.responses.read),
```

Comment:
I believe what we want here is
```ts
[
  r.ProgressReportCommunityStory,
  r.ProgressReportHighlight,
  r.ProgressReportTeamNews,
  r.ProgressReportVarianceExplanation,
].flatMap((it) => it.when(member).read),
```

Your lines 26 is trying to be the same thing as 29-31. So we don't need both, and we prefer the non-children syntax for static parent cases.

The other difference is that 29-31 allows reading globally, instead of only if a member. Which I don't think is the intent.

Also I know we talked before about this, but the
```ts
.specifically((p) => p.responses.read),
```
shouldn't be needed as it's the same action permission as what the object level has.